chain/store/store.go

```diff
@@ -11,7 +11,8 @@ import (
 	"sync"
 	"time"
 
-	lru "github.com/hashicorp/golang-lru/v2"
+	"github.com/hashicorp/golang-lru/arc/v2"
```

Comment:
golang-lru introduced a breaking change in the v2.0.3 patch release. See https://github.com/hashicorp/golang-lru/issues/142 for discussion. The author has his own interpretation of the semver specification, that diverges from everyone else's reading, and seems oblivious to the amount of pain his actions cause in the ecosystem.